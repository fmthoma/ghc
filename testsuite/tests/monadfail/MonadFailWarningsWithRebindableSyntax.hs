{-# LANGUAGE RebindableSyntax #-}

module MonadFailWarningsWithRebindableSyntax where

import Prelude

test1 f g = do
    Just x <- f
    g
