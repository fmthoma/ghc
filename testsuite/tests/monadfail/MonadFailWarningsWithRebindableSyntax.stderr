
MonadFailWarningsWithRebindableSyntax.hs:8:5: warning:
    The failable pattern ‘Just x’
      is used together with -XRebindableSyntax. If this is intentional,
      compile with -fno-warn-missing-monadfail-instance.
